### PR TITLE
Add package to all distros

### PR DESCRIPTION
**Pull request by ekohl**
When adding a package, it is common to use the EL7 tito tag. This makes sure it is added to all distros. This mostly means EL8 but for client distros it includes Fedora.

### CODE REVIEW

File: add_gem_package.sh [hunk 1]
```diff
@@ -49,34 +49,74 @@ generate_gem_package() {
 	popd
 }
 
+add_to_all_tito_props() {
+	add_to_tito_props $TITO_TAG
+	if [[ $TITO_TAG == "foreman-nightly-rhel7" ]] || [[ $TITO_TAG == "foreman-nightly-nonscl-rhel7" ]] ; then
```
Comment (evgeni) [inline add_gem_package.sh:54]: should this instead match `$TITO_TAG == foreman-nightly-*`?

File: add_gem_package.sh [hunk 2]
```diff
@@ -49,34 +49,74 @@ generate_gem_package() {
...
 	local whitelist=$(echo "$current_whitelist $PACKAGE_NAME" | tr " " "\n" | sort -u)
-	crudini --set rel-eng/tito.props $TITO_TAG whitelist "$whitelist"
+	crudini --set rel-eng/tito.props $tag whitelist "$whitelist"
 	export LC_COLLATE=$original_locale
 	git add rel-eng/tito.props
 }
 
+add_gem_to_all_comps() {
+	add_gem_to_comps $TITO_TAG
+	if [[ $TITO_TAG == "foreman-nightly-rhel7" ]] || [[ $TITO_TAG == "foreman-nightly-nonscl-rhel7" ]] ; then
```
Comment (evgeni) [inline add_gem_package.sh:82]: same q as above

**Review by evgeni — APPROVED** [review]
one q, but nothing that should prevent merging

File: add_gem_package.sh (hunk 1 above)
Comment (ekohl) [reply add_gem_package.sh:54]: Doesn't that mean that if you pass EL8 as a tito tag it gets added twice?

Comment (evgeni) [reply add_gem_package.sh:54]: Yeah, that'd mean it. But shouldn't the underlying "add" code handle that?

Anyways, that's not blocking, just an idea ;)

Comment (ekohl) [reply add_gem_package.sh:54]: I thought about that, but I'd prefer to merge this now and improve it if people dislike it. I was mostly annoyed at having to add it manually to additional files and we've seen the same thing in reviews.

**Pull request merged**